### DISCUSSION TITLE
在visual studio code設定auto git fetch以及週期

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS3073-C/Python-Community-GS3073-C#278] [General]
根據下圖操作就可以喔~

![image](https://user-images.githubusercontent.com/1768891/157214371-a8146bcd-46fa-47c5-95d5-c6b37a7af0d9.png)

![image](https://user-images.githubusercontent.com/1768891/157214773-f28e0591-b555-41b9-9442-197e151e004e.png)

![image](https://user-images.githubusercontent.com/1768891/157218310-d37266f6-01a8-4427-8415-39af76b258c8.png)

**Comment by enoch-lightning** [comment]
我把操作老師`在visual studio code設定auto git fetch以及週期`的過程錄下來，講一些我的理解，因為我不熟git fetch。
[在visual studio code設定auto git fetch以及週期](https://youtu.be/rREenNuW3T4)
